pkg/controller/sparkapplication/job.go

```diff
@@ -30,62 +35,96 @@ import (
 	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/config"
 )
 
+const (
+	sparkSubmitPodMemory = "1000Mi"
+	sparkSubmitPodCpu    = "1024m"
+)
+
+type submissionJobManagerIface interface {
+	createSubmissionJob(app *v1beta1.SparkApplication) (*string, *string, error)
```

Comment:
Please change the return types to `string, string, error` instead.

Reply:
done